website/app-templates/smarty/pages/home.tpl

```diff
@@ -38,10 +41,14 @@
     </tbody>
   </table>
 </div>
+{else}{* no recent moves means db is empty *}
```

Comment:
If `else` is not used please remove, else add a message targeted to the developer.